.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     'import/no-unresolved': 'error',
     // Since React 17 and typescript 4.1 you can safely disable the rule
     'react/react-in-jsx-scope': 'off',
+    curly: ['error', 'all'],
```

Comment:
should we put quotes around curly for consistency?

Reply:
i did originally, but either prettier or eslint automatically changed it to that lol